### PR TITLE
Farsi Translation

### PR DESCRIPTION
**Pull request by silvercover**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by gallexi** [comment]
Thank you for working on this @silvercover ! I'll convert this to a draft for now while it's in progress, but feel free to convert it back when you're finished!

**Comment by ArtNow198** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Comment by loida24** [comment]
Acces

**Review by loida24 — COMMENTED** [review]
☺️

**Review by loida24 — COMMENTED** [review]
☺️

**Review by loida24 — DISMISSED** [review]

**Comment by NicoleAnn926** [comment]
Hi. Thanks for the emails and invitation to be a part of this. I have a lot of questions and I have a lot to learn about all of this. Tomorrow morning I'll go through the documents that GitHub offers to those of us who are beginners with coding. I'm clueless as to why so many people have been working on my phone. Mozilla and Google developers as well as USDA, National Science Foundation, Stacked, NIST, CIS.... I don't know what to think. Because of the many experiments logged in my phone and the death threats to be a live subject in the the entertainment business ownership of people. I've been overwhelmed with all of this and I would like to know what is going on. This hasn't exactly been a fair set-up, as I'm outnumbered and deemed the "enemy" from the other people who have focused on my life with constant contact within their own agenda for me trapped in this scandal.

**Comment by ErdaKamarudin** [comment]
Code

**Review by SaintPatricksCoin — DISMISSED** [review]
1619798549686_1954324047_ca5176a7 gif

**Review by SaintPatricksCoin — APPROVED** [review]

**Review by SaintPatricksCoin — DISMISSED** [review]

**Comment by silvercover** [comment]
> Thank you for working on this @silvercover ! I'll convert this to a draft for now while it's in progress, but feel free to convert it back when you're finished!

I've done the whole translation. please undo draft status. also I want to mention Farsi is RTL language and I think some tweaks and style editing needed.

**Comment by silvercover** [comment]
> ☺️

Would you please review my pull request?

**Review by Mohsen-Phoenix — APPROVED** [review]

**Comment by silvercover** [comment]
> ☺️

Would you please review my pull request?

**Review by noncash15 — APPROVED** [review]

**Comment by itsbagpack** [comment]
Thank you for the translation work! I'll merge this when the tests pass.

**Comment by itsbagpack** [comment]
@silvercover Would you mind trying to merge or rebase latest master into your PR? Hopefully the tests will get triggered by that. Thank you!

**Comment by silvercover** [comment]
@itsbagpack Thanks for your help. please see this:
Capture

what should I do then?

**Comment by itsbagpack** [comment]
@silvercover Hi, as per my earlier comments and your screenshot, the build is stuck. As such, I suggest merging latest master into your branch to hopefully kick off a new build.

**Comment by ghasrfakhri** [comment]
@itsbagpack @silvercover Hi, I just merged the latest master. I hope it helps.

**Review by MaryJaneLove4 — APPROVED** [review]
Da

**Review by MaryJaneLove4 — APPROVED** [review]
Schvaleno

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]
Ok

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]
Cool

**Review by SaintPatricksCoin — APPROVED** [review]
@SaintPatricksCoin 
One Second...

1619798549686_1954324047_ca5176a7 gif

**Comment by MikeMcQuaid** [comment]
@silvercover What's the latest status of this? Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Awaiting CI fixes

**Comment by silvercover** [comment]
> @silvercover What's the latest status of this? Thanks!

Capture

Hi, I get these messages and don't know what to do.

**Comment by MikeMcQuaid** [comment]
@silvercover I have to manually enable CI when you push new commits. There's another error here: https://github.com/github/opensource.guide/pull/2351/checks?check_run_id=3554172773#step:7:29

**Comment by silvercover** [comment]
@MikeMcQuaid 
as for [this](https://github.com/github/opensource.guide/pull/2351/checks?check_run_id=3554172773#step:7:29), we've reviewed it again and found nothing to fix.

I don't know what control characters caused this.

**Review by ErdaKamarudin — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — APPROVED** [review]

**Comment by jova124** [comment]
Jovan Ramirez

lunes, 13 de septiembre 2021, 1:09 +03:00 de MaryJaneLove4 ***@***.***>:
***@***.*** approved this pull request.

**Comment by MikeMcQuaid** [comment]
@silvercover You can push new commits now and it'll retest automatically. I'm afraid I can't merge until we figure out the failures here 😭

**Comment by silvercover** [comment]
@MikeMcQuaid Hopefully I fixed few errors and warnings and now there is only 1 warning which I really don't have any clue about it in this file:
[how-to-contribute.md](https://github.com/parscoders/opensource.guide/blob/fa/_articles/fa/how-to-contribute.md)

  _`137:1-137:57  warning  Missing blank line before block node  no-missing-blank-lines  remark-lint`_

File: _articles/fa/how-to-contribute.md
Comment (MikeMcQuaid) [inline _articles/fa/how-to-contribute.md:138]:
```suggestion

### لازم نیست فقط روی پروژه‌های نرم‌افزاری وقت صرف کنید!

```

Comment (MikeMcQuaid) [reply _articles/fa/how-to-contribute.md:138]: @silvercover This should fix the CI failure you mentioned

**Comment by silvercover** [comment]
@MikeMcQuaid All checks have passed!
🥳

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @silvercover!

**Comment by silvercover** [comment]
@MikeMcQuaid Thank you.

Since Farsi is RTL language, what's your suggestion on making this RTL. I don't know whether putting `<div dir='rtl'>`  at the beginning allowed or not.

**Comment by MikeMcQuaid** [comment]
@silvercover you're welcome!

I'd suggest something dynamic/conditional like https://github.com/github/opensourcefriday/blob/7ce1cdc59edc6840b390c2d02259878fb8c9c606/app/views/layouts/application.html.erb#L2 should be added (but will use Liquid rather than ERB). Could you open a PR for discussion?

**Review by Scheusal13 — COMMENTED** [review]
0xEa5C920973eBfff95270dd52c8045c4e6eBFc847

**Pull request merged**
